### PR TITLE
feat(FEC-9023): More logger options

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

add configuration logger options.
handler to output logs with customize function.
option to remove playerVersion from log.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: flow-typed/types/log-level.js [hunk 1]
```diff
@@ -2,3 +2,9 @@
 declare type PKLogLevelObject = { value: number, name: string };
 declare type PKLogLevels = { [level: string]: PKLogLevelObject };
 declare type PKLogLevelTypes = { [level: string]: string };
+declare type LogHandlerType = (messages: any[], context: Object) => void;
+declare type PKLogConfigObject = {
+  level:string,
+  playerVersion:boolean,
+  handler:LogHandlerType
```
Comment (yairans) [inline flow-typed/types/log-level.js]: I'm not flow but seems these should be `?boolean` and `?LogHandlerType`   :)

File: flow-typed/types/log-level.js [hunk 2]
```diff
@@ -2,3 +2,9 @@
 declare type PKLogLevelObject = { value: number, name: string };
 declare type PKLogLevels = { [level: string]: PKLogLevelObject };
 declare type PKLogLevelTypes = { [level: string]: string };
+declare type LogHandlerType = (messages: any[], context: Object) => void;
+declare type PKLogConfigObject = {
+  level:string,
+  playerVersion:boolean,
```
Comment (yairans) [inline flow-typed/types/log-level.js]: `playerVersion` is used in kaltura player only so move it there (declare it in kaltura player and use the playkit as base type and add `playerVersion`. see https://github.com/kaltura/playkit-js-providers/blob/master/flow-typed/types/media-info.js for example)

Comment (Yuvalke) [reply flow-typed/types/log-level.js]: fixed

File: flow-typed/types/log-level.js (hunk 1 above)
Comment (Yuvalke) [reply flow-typed/types/log-level.js]: fixed

**Pull request merged**